src/actions/send.js

```diff
@@ -0,0 +1,40 @@
+import { createActions } from 'redux-actions';
+
+import { showAlert } from '../utils/alerts';
+import { wallet } from '../utils/wallet';
+
+export const transfer = ({ contractName, amount, memo, receiverId, isStorageBalanceAvailable }) => async (dispatch, gesState) => {
+    if (contractName && !isStorageBalanceAvailable) {
+        await dispatch(send.transfer.storageDeposit(contractName, receiverId));
+    }
+
+    const { transaction, status } = await dispatch(send.transfer[contractName ? 'TOKENS' : 'NEAR'](contractName, amount, memo, receiverId));
```

Comment:
This function signature isn't actually valid for `wallet.sendMoney()`, which is what is selected when `contractName` is not set here.  See above comment around separating logic for different types of sends; trying to do too much in a single stream of logic leads to these types of bugs and mismatches in logic...